wortell/AZSentinel
# Added function to get the alert list templates provided by Microsoft

Added the Get-AzSentinelAlertRuleTemplates function to get alert rules list templates provided by Microsoft. It can be useful to keep an automatic tracking of new templates deployed into Sentinel.


# Summary of the Pull Request

...


## References

...


## PR Checklist

**By submitting this pull request, I confirm the following:**

*please fill any appropriate checkboxes, e.g: [X]*

- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] It is compatible with the [MIT License](https://opensource.org/licenses/MIT)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---


## Detailed Description of the Pull Request / Additional comments

New function called Get-AzSentinelAlertRuleTemplates has been developed in order to try to use when new alert rule templates can be deployed by microsoft into Sentinel to start using them.

![image](https://user-images.githubusercontent.com/34833071/85732017-4e01aa80-b6fb-11ea-84de-5036ef84f5ed.png)



## Validation Steps Performed

Validation steps were the use of debugger to check step by step each ways in the code.

## How does this PR accomplish the above

New Functionality added to try to deploy some tool for automatic detection when rew rules would be added into Sentinel alert rules template.

## What documentation changes (if any) are needed to support this PR

Not needed any change to use the new functionality

---

* You must follow the template instructions. Failure to do so will result in your pull request being closed.
